CoseSign1/DetachedSignatureFactory.cs

```diff
@@ -416,9 +426,9 @@ public DetachedSignatureFactory(HashAlgorithmName hashAlgorithmName, ICoseSign1M
     }
 
     /// <summary>
-    /// quick lookup of algorithm name based on hash size
+    /// quick lookup of algorithm name based on size of raw hash in bytes

```

Comment:
Can be helpful to leave a link to the source of this information for the next maintainer.